tests/source-maps.test.js

```diff
@@ -38,33 +38,34 @@ test.skip('apply generates source maps', async () => {
   // All CSS generated by Tailwind CSS should be annotated with source maps
   // And always be able to point to the original source file
   expect(sources).not.toContain('<no source>')
-  expect(sources.length).toBe(1)
+  expect(sources.length).toBe(2)
```

Comment:
All of these are increased by 1. If you (temporarily) set the `minify: false` to `minify: true` in the lightningCssPlugin then it goes down by 1 again.

Reply:
the number of sources should definitely just be `1`

Introducing additional sources that disappear during minification sounds like a lightningcss bug. I'll set up an isolated test case for that and file a bug report.